cicd/tests/testcases/am-smoke.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+################################################################################
+# ForgeOps AM smoke tests definitions
+#
+# This suite cointains a set of smoke tests that are intended to be run
+# as part of CI/CD process.
+#
+# To run this tests, run ../forgeops-smoke-tests.sh -s am_smoke.sh
+#
+# Each testcase is going to be described inline
+#
+# Each test must set PASS=true/false and optionally RES with content that
+# will be logged in out.txt
+#
+################################################################################
+
+# Testcase definitions
+
+# Ping test - Touch AM isAlive.jsp endpoint and expect HTTP-200
+ping_test() {
+  TESTNAME="AM ping test"
+  RES=$(curl -v -LI $AM_URL/isAlive.jsp \
+          -o /dev/null -w '%{http_code}\n' -s)
+  if [ "$RES" = "200" ]; then
+    PASS=true
+  fi
+}
+
+# Admin login - Login into AM as Amadmin, expect tokenId to be present in resp.
+admin_login_test() {
+  TESTNAME="AM Login test"
+  RES=$( curl --request POST --header \
```

Comment:
I usually change the Authentication->Settings->Organization Authentication Configuration to point to my own service.  This means that you can no longer login into AM as "amadmin" using standard json/authenticate endpoint.  You have to use something like json/authenticate/?authIndexType=service&authIndexValue=adminconsoleservice.  So lets make it a habit to call the endpoitn with these query params as a standard practice in our scripts

Reply:
In that case we will need to make this change to default AM config as well. I will speak to Lee if it's OK to change it. 

This request is the same as XUI does, right ?